### PR TITLE
HCAP-1140 | Manage Reject Status for Multi Org

### PR DESCRIPTION
**Pull request by fwpushan**
For Multi-Org reject (Archive from **My Candidate**), implementing as user-specific reject.

### CODE REVIEW

**Review by dbayly-freshworks — APPROVED** [review]
Looks good.

**Review by taraepp — APPROVED** [review]
Looks good to me

**Review by alva-fresh — APPROVED** [review]

File: server/package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "eslint-plugin-import": "2.22.0",
     "eslint-plugin-jest": "23.20.0",
     "eslint-plugin-prettier": "3.4.0",
-    "jest": "25.3.0",
+    "jest": "28.0.2",
```
Comment (alva-fresh) [inline server/package.json:62]: Why is it necessary to update `jest` version?

Comment (fwpushan) [reply server/package.json:62]: This is result of audit fix

**Pull request merged**